.eslintrc.js

```diff
@@ -22,15 +22,30 @@ const config = {
       './www/tsconfig.json',
     ], // Allows for the use of rules which require parserServices to be generated
   },
+  // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
   rules: {
-    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    // These rules aren't enabled in typescript-eslint's basic recommended config, but we like them
+    '@typescript-eslint/no-non-null-assertion': 'error',
+
+    // These rules enabled in typescript-eslint's configs don't apply here
+    '@typescript-eslint/consistent-indexed-object-style': 'off',
+    '@typescript-eslint/consistent-type-definitions': 'off',
+    '@typescript-eslint/no-empty-interface': 'off',
+    '@typescript-eslint/sort-type-constituents': 'off',
+
+    // Todo: do we want these?
+    '@typescript-eslint/array-type': 'off',
+    '@typescript-eslint/class-literal-property-style': 'off',
+    '@typescript-eslint/dot-notation': 'off',
+    '@typescript-eslint/no-confusing-void-expression': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/prefer-for-of': 'off',
+    '@typescript-eslint/prefer-function-type': 'off',
+    '@typescript-eslint/prefer-nullish-coalescing': 'off',
```

Comment:
Each of these had at least a few existing warnings, so I didn't want to overreach and solve for them without asking. Are any of these rules ones you feel particularly strongly about?

https://typescript-eslint.io/rules/array-type
https://typescript-eslint.io/rules/class-literal-property-style
https://typescript-eslint.io/rules/dot-notation
https://typescript-eslint.io/rules/no-confusing-void-expression
https://typescript-eslint.io/rules/no-explicit-any
https://typescript-eslint.io/rules/prefer-for-of
https://typescript-eslint.io/rules/prefer-function-type
https://typescript-eslint.io/rules/prefer-nullish-coalescing

If not, I can go ahead and fix for all their complaints.

Reply:
We need `any`s in the source code, apart from that I'm fine with swapping I think, cc @sachinraja

Reply:
I'm good with all of them